setup-server.js

```diff
@@ -1,10 +1,27 @@
+/**
+ * You should not need to change this
+ */
+
 const express = require('express');
 const path = require('path');
 const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+const fs = require('fs');
+dotenv.config();
```

Comment:
clever, where did you get that from?

Reply:
This is from the setup that we got yesterday.